mopidy_youtube/apis/youtube_bs4api.py

```diff
@@ -217,26 +235,15 @@ def list_playlistitems(cls, id, page, max_results):
                 logger.info(
                     "nothing in the soup, trying japi list_playlistitems"
                 )
-                json_regex = r'window\["ytInitialData"] = ({.*?});'
-                extracted_json = re.search(json_regex, result.text).group(1)
-                result_json = json.loads(extracted_json)["contents"][
-                    "twoColumnBrowseResultsRenderer"
-                ]["tabs"][0]["tabRenderer"]["content"]["sectionListRenderer"][
-                    "contents"
-                ][
-                    0
-                ][
-                    "itemSectionRenderer"
-                ][
-                    "contents"
-                ][
-                    0
-                ][
-                    "playlistVideoListRenderer"
-                ][
-                    "contents"
-                ]
-                items = jAPI.json_to_items(cls, result_json)
+
+                yt_data = cls._find_yt_data(result.text)
+                json = yt_data["contents"]\
```

Comment:
Calling this `json` breaks things; better to call it result_json or similar, as previously, I think.